plenum/common/constants.py

```diff
@@ -57,6 +57,15 @@
 BLS_PREFIX = "BLS: "
 OBSERVER_PREFIX = "OBSERVER: "
 
+ID = "id"
+TAG = "tag"
+CRED_DEF_ID = "credDefId"
+ISSUANCE_TYPE = "issuanceType"
+MAX_CRED_NUM = "maxCredNum"
+PUBLIC_KEYS = "publicKeys"
+TAILS_HASH = "tailsHash"
+TAILS_LOCATION = "tailsLocation"
+SUBMITTER_DID = "submitterDid"
```

Comment:
ID and SUBMITTER_DID can be here, but SUBMITTER_DID is not going to be used right now (identifier will be used instead)